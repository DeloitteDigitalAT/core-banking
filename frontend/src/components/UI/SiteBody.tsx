import React from "react";

const SiteBody: React.FC = properties => {
  return <div className="max-w-6xl mx-auto px-2 md:px-4 flex flex-col min-h-screen">{properties.children}</div>;
};

export default SiteBody;
