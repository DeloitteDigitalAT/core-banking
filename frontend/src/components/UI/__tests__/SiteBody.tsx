import React from "react";
import renderer from "react-test-renderer";
import SiteBody from "../SiteBody";

describe("UI/SiteBody component", () => {
  test("snapshot renders", () => {
    const tree = renderer.create(<SiteBody />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
