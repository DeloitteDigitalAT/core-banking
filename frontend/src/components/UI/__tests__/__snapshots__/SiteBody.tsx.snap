// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/SiteBody component snapshot renders 1`] = `
<div
  className="max-w-6xl mx-auto px-2 md:px-4 flex flex-col min-h-screen"
/>
`;
